packages/chakra-ui-docs/docs/tabs.mdx

```diff
@@ -1 +1,348 @@
-# Tabs
\ No newline at end of file
+import SEO from '../components/SEO'
+
+<SEO
+  title="Tabs"
+  description="The Tab component consists of clickable tabs, that are aligned side by side."
+/>
+
+# Tabs
+
+An accessible tabs component.
+
+The `Tab` and `TabPanel` elements are associated by their order in the tree.
```

Comment:
`CTab` and `CTabPanel`. Then this shall be good to go!

Reply:
@codebender828 resolved!